src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import Weather from "./Weather";
+import "./App.css";
+
+class App extends Component {
+  state = {
+    background: "",
+  };
+
+  componentDidMount() {
+    fetch("https://source.unsplash.com/1600x900/?nature,background")
+      .then((res) => this.setState({ background: res.url }))
+      .catch((err) => {
+        console.log(err);
```

Comment:
Consider having a 'fallback' background that could be used in the event of an api error, console.logs won't be very helpful to the user.